src/uwtools/drivers/fv3.py

```diff
@@ -217,33 +217,10 @@ def runscript(self):
 
     # Private workflow tasks
 
-    @external
-    def _file(self, path: Path):
-        """
-        An existing file.
-
-        :param path: Path to the file.
-        """
-        yield "File %s" % path
-        yield asset(path, path.is_file)
-
-    @task
-    def _filecopy(self, src: Path, dst: Path):
-        """
-        A copy of an existing file.
-
-        :param src: Path to the source file.
-        :param dst: Path to the destination file to create.
-        """
-        yield "Copy %s -> %s" % (src, dst)
-        yield asset(dst, dst.is_file)
-        yield self._file(src)
-        copy(src, dst)
```

Comment:
These two private (task) methods, along with `_symlink()` below, now live in `uwtools.utils.tasks` as public functions.